samples/emoji-search/presenters/src/commonMain/kotlin/app/cash/zipline/samples/emojisearch/EmojiSearchPresenter.kt

```diff
@@ -19,7 +19,7 @@ import app.cash.zipline.ZiplineService
 import kotlinx.coroutines.flow.Flow
 
 interface EmojiSearchPresenter : ZiplineService {
-  suspend fun produceModels(
+  fun produceModels(
```

Comment:
This was unnecessary and it also triggered this bug: https://github.com/cashapp/zipline/issues/654